Logic/item_detection.py

```diff
@@ -0,0 +1,196 @@
+'''Item Detection Module'''
+
+import os
+import numpy as np
+import cv2
+
+from skimage.measure import compare_ssim as ssim
+
+from Config.eight_ball_lookup import EightBallLookup
+
+class ItemDetection:
+    '''Responsible for detecting game balls and holes'''
+
+    lookup = EightBallLookup()
+
+    def identify_parameters(self, for_holes, for_balls):
+        '''Responsible iterating through a number of images and saving the image results based on different parameter values'''
+
+        for ball_folder in os.listdir(self.lookup.training_path):
```

Comment:
ball_folder might be misleading, consider, ball_image or ball_file

Reply:
Done